hoot-core/src/main/cpp/hoot/core/MapProjector.cpp

```diff
@@ -71,39 +71,32 @@ class DisableCplErrors
   static void emptyErrorHandler(CPLErr, int, const char *) { }
 };
 
-class ReprojectCoordinateFilter : public CoordinateFilter
```

Comment:
Why move this class from the .cpp to the .h? This class is only used inside MapProjector so I don't think there is any harm in leaving it there. Keeps the header file smaller/cleaner.